### DISCUSSION TITLE
JDBI 3.42.0

### DISCUSSION THREAD

**Discussion by hgschmie** [jdbi/jdbi#2550] [Announcements]
- Add Spring Jdbi repositories with `@EnableJdbiRepositories`, thanks @xfredk (#2528, #2544)
- Support Kotlin coroutine scope. (#2524) - Suggested by @anderssv on the mailing list
- Move from Kotlin 1.5 to 1.6 as 1.5 is deprecated and will be removed.
- Add support for Function arguments, similar to Consumer arguments, to SQL objects (#2326)
- Fix kotlin deprecation warnings (#2511) - Thanks @lwach-allegro for contributing a PR
- lexer fixes (#2520), this should fix all problems with identifier names (fixes #2481, #2499 and #2510)
- correctness fix for Handle creation (#2541)
- Fix Vavr argument usage in SqlObjects (reported by @diversit, #2529)
- Fix MySQL Script parsing (reported by @IrinaTerlizhenko, #2535)
- Added a written [security policy](https://github.com/jdbi/jdbi/blob/master/SECURITY.md).

<hr /><em>This discussion was created from the release <a href='https://github.com/jdbi/jdbi/releases/tag/v3.42.0'>JDBI 3.42.0</a>.</em>

**Comment by d2k2-git** [comment]
when using @JdbiRepository on a Dao interface, we dont need to create Dao beans like this anymore?

```java
@Configuration
public class DaoConfiguration {

	@Bean
	PersonDao personDao(Jdbi jdbi) {
		return jdbi.onDemand(PersonDao.class);
	}
}
```
